src/Apps/Artist/Routes/WorksForSale/Components/ArtistArtworkFilter.tsx

```diff
@@ -67,16 +68,22 @@ const ArtistArtworkFilter: React.FC<ArtistArtworkFilterProps> = props => {
       ZeroState={ZeroState}
       userPreferredMetric={userPreferences?.metric}
     >
-      <ArtworkFilterSavedSearchAlertContextProvider entity={savedSearchEntity}>
-        <BaseArtworkFilter
-          relay={relay}
-          viewer={artist}
-          Filters={<ArtistArtworkFilters />}
-          relayVariables={{
-            aggregations: ["TOTAL"],
-          }}
-        />
-      </ArtworkFilterSavedSearchAlertContextProvider>
+      <ArtworkFilterAlertContextProvider
+        initialCriteria={{ artistIDs: [artist.internalID] }}
+      >
```

Comment:
We're wrapping here because there are quite a few components nested under this point in the component tree that render a Create Alert button and each needs to be able to dispatch the modal.